packages/rebber-plugins/src/preprocessors/katexConstants.json

```diff
@@ -1 +1 @@
bk","\\llap","\\rlap","\\clap","\\mathstrut","\\underbar","\\not","\\neq","\\ne","\\notin","\\ulcorner","\\urcorner","\\llcorner","\\lrcorner","\\vdots","\\varGamma","\\varDelta","\\varTheta","\\varLambda","\\varXi","\\varPi","\\varSigma","\\varUpsilon","\\varPhi","\\varPsi","\\varOmega","\\substack","\\colon","\\boxed","\\iff","\\implies","\\impliedby","\\dots","\\dotso","\\dotsc","\\cdots","\\dotsb","\\dotsm","\\dotsi","\\dotsx","\\DOTSI","\\DOTSB","\\DOTSX","\\tmspace","\\,","\\thinspace","\\>","\\:","\\medspace","\\;","\\thickspace","\\!","\\negthinspace","\\negmedspace","\\negthickspace","\\enspace","\\enskip","\\quad","\\qquad","\\tag","\\tag@paren","\\tag@literal","\\bmod","\\pod","\\pmod","\\mod","\\pmb","\\newline","\\TeX","\\LaTeX","\\KaTeX","\\hspace","\\@hspace","\\@hspacer","\\ordinarycolon","\\vcentcolon","\\dblcolon","\\coloneqq","\\Coloneqq","\\coloneq","\\Coloneq","\\eqqcolon","\\Eqqcolon","\\eqcolon","\\Eqcolon","\\colonapprox","\\Colonapprox","\\colonsim","\\Colonsim","\\ratio","\\coloncolon","\\colonequals","\\coloncolonequals","\\equalscolon","\\equalscoloncolon","\\colonminus","\\coloncolonminus","\\minuscolon","\\minuscoloncolon","\\coloncolonapprox","\\coloncolonsim","\\simcolon","\\simcoloncolon","\\approxcolon","\\approxcoloncolon","\\notni","\\limsup","\\liminf","\\injlim","\\projlim","\\varlimsup","\\varliminf","\\varinjlim","\\varprojlim","\\gvertneqq","\\lvertneqq","\\ngeqq","\\ngeqslant","\\nleqq","\\nleqslant","\\nshortmid","\\nshortparallel","\\nsubseteqq","\\nsupseteqq","\\varsubsetneq","\\varsubsetneqq","\\varsupsetneq","\\varsupsetneqq","\\imath","\\jmath","\\llbracket","\\rrbracket","\\lBrace","\\rBrace","\\minuso","\\darr","\\dArr","\\Darr","\\lang","\\rang","\\uarr","\\uArr","\\Uarr","\\N","\\R","\\Z","\\alef","\\alefsym","\\Alpha","\\Beta","\\bull","\\Chi","\\clubs","\\cnums","\\Complex","\\Dagger","\\diamonds","\\empty","\\Epsilon","\\Eta","\\exist","\\harr","\\hArr","\\Harr","\\hearts","\\image","\\infin","\\Iota","\\isin","\\Kappa","\\larr","\\lArr","\\Larr","\\lrarr","\\lrArr","\\Lrarr","\\Mu","\\natnums","\\Nu","\\Omicron","\\plusmn","\\rarr","\\rArr","\\Rarr","\\real","\\reals","\\Reals","\\Rho","\\sdot","\\sect","\\spades","\\sub","\\sube","\\supe","\\Tau","\\thetasym","\\weierp","\\Zeta","\\argmin","\\argmax","\\plim","\\bra","\\ket","\\braket","\\Bra","\\Ket","\\angln","\\blue","\\orange","\\pink","\\red","\\green","\\gray","\\purple","\\blueA","\\blueB","\\blueC","\\blueD","\\blueE","\\tealA","\\tealB","\\tealC","\\tealD","\\tealE","\\greenA","\\greenB","\\greenC","\\greenD","\\greenE","\\goldA","\\goldB","\\goldC","\\goldD","\\goldE","\\redA","\\redB","\\redC","\\redD","\\redE","\\maroonA","\\maroonB","\\maroonC","\\maroonD","\\maroonE","\\purpleA","\\purpleB","\\purpleC","\\purpleD","\\purpleE","\\mintA","\\mintB","\\mintC","\\grayA","\\grayB","\\grayC","\\grayD","\\grayE","\\grayF","\\grayG","\\grayH","\\grayI","\\kaBlue","\\kaGreen","\\ce", "\\\\", "\\nabla", "\\{", "\\}", "\\,", "\\[", "\\}"]
```

Comment:
All the commands you added already exist in the file (check before with a quick search), some of them even appear twice. I will regenerate the constants file before releasing to remove the duplicates, but can you remove yours already?

The only exception (symbol not already in the file) is the `\[` which doesn't seem to be parsed by KaTeX (from what I can see in the live demo), so no changes should be required on this file.